robot_controllers/src/pid.cpp

```diff
@@ -157,8 +157,8 @@ double PID::update(double error, double dt)
   {
     error_dot = (error-error_last_)/dt;
   }
-  return update(error, error_dot, dt);
   error_last_ = error;
+  return update(error, error_dot, dt);
```

Comment:
@dbking77 -- thoughts on this? Should we only update error_last_ when dt > 0? or every time? (it appears that controls_toolbox chooses to only update when dt > 0